src/pages/events.js

```diff
@@ -45,36 +48,88 @@ const dallasLogoStyles = {
 };
 
 // Component Definition
-const IndexPage = () => {
-  return (
-    <Layout>
-      <div css={rootStyles}>
-        <section css={heroContainerStyles}>
-          <header>
-            <h1 css={heroTitleStyles}>Events</h1>
-            <div css={heroTextStyles}>Where DFW meets</div>
-          </header>
-        </section>
-
-        <section css={contentStyles}>
-          <h3>December 2018</h3>
-          <h4>ReactJS @ BottleRocket</h4>
-          <div>6:15 - 7:00pm: Pizza/Drinks & meeting other people</div>
-          <div>6:30 - 7:00pm: Jobs Open Mic and jobs follow-up</div>
-          <div>7:00 - 7:45pm - Morgan Dedmon - "WASM: What is that?"</div>
-          <div>8:00 - 8:45pm: Salvador Aceves - "Redux Sagas in Practice"</div>
-
-          <p style={{ marginTop: 12 }}>
-            <a href="https://www.meetup.com/ReactJSDallas/events/pbbdwnyxqbpb/">Meetup link</a>
-          </p>
-        </section>
-
-        <div css={dallasLogoContainerStyles}>
-          <DallasLogoSvg style={dallasLogoStyles} />
-        </div>
-      </div>
-    </Layout>
-  );
-};
+export default class IndexPage extends React.Component {
+  
+  //Setup default state for our index page
+  constructor(props)
+  {
+    super(props);
+    this.state = {
+      events: [] //we will fill this after fetch
+    };
+
+
+  }
+
+  //Fetches from meetup API and stores our events
+  componentDidMount ()
+  {
+    //console.log('mounted');
+
+    //Fetch all events from our React JS meetup group!
+    fetch("https://api.meetup.com/reactjsdallas/events?key=1570558171e5f7833a5f7b1d682813", {
+      "credentials":"include",
+      "headers":{
+        "accept":"text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8",
+        "accept-language":"en-US,en;q=0.9",
+        "cache-control":"max-age=0",
+        "if-none-match":"\"a668e670716ce826669d946fc0f0c947-gzip\"",
+        "upgrade-insecure-requests":"1"
+      },
+      "referrerPolicy":"no-referrer-when-downgrade",
+      "body":null,
+      "method":"GET",
+    })
+    .then(res => res.json())
+    .then(events => {
+      //Log our response
+      //console.log(events);
+
+      const choppedEvents = events.slice(0,3);
 
-export default IndexPage;
+      //Set our events to the state (forces page refresh)
+      this.setState({
+        events: choppedEvents
+      });
+
+    })
+    .catch(err=>console.log(err));
+  } 
+
+
+  //Rendering of events page
+  render () {
+    //console.log('rendering');
+
+    //Create our Event elements from the first 3 events
+    const eventElements = this.state.events.map((event, index) =>{
+      return <Event 
```

Comment:
We will need a `key` prop passed into `<Event>` here. It should be unique. Also, `index` is not used, so it can be safely removed.